coral/src/app/layout/main-navigation/main-navigation-utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Routes } from "src/app/router_utils";
+import { DEV_MODE } from "src/constants";
+
+function createNavLink(path: Routes) {
```

Comment:
I wonder if this util would not be better higher up, in `router_utils`. Then the routes enum would have entries like this (if we rename it to `createLink` too, it's for all `coral` routes, right, not only nav lnks?)
```
  TOPICS = createLink("/topics"),
```

Reply:
I'm now realising we don't need to function at all because we have 'useRouter` as prob  now 🤦  which probably makes it completely unnecessary. 😅

Reply:
Nope, not needed. 🤦‍♀️ 🤦‍♀️